usb/esp_tinyusb/Kconfig

```diff
@@ -259,12 +259,15 @@ menu "TinyUSB Stack"
             help
                 Select network driver you want to use.
 
-            config TINYUSB_NET_MODE_ECM_RNDIS
-                bool "ECM/RNDIS"
-
             config TINYUSB_NET_MODE_NCM
                 bool "NCM"
 
+            config TINYUSB_NET_MODE_RNDIS
+                bool "RNDIS"
+
+            config TINYUSB_NET_MODE_ECM
+                bool "ECM"
+
```

Comment:
Can we keep it tha way it was? ECM+RNDIS together? There are 2 reasons for this:

1. The TinyUSB driver has ECM+RNDIS protocol together
2. We usually don't want to enable ECM and disalble RNDIS (or vice versa). RNDIS works with windows, while ECM workds with linux and macOS. Only with both we can support all operating systems